### ISSUE TITLE
Preference to automatically copy to clipboard after transforming text

### ISSUE THREAD

**Issue by jpollone** [IvanMathy/Boop#147]
I like this idea of saving two keystrokes by allowing an option to copy text to the clipboard after a single command is executed.

Here's a sample flow:

1. Select a piece of text from my web browser's text area input
2. Switch to Boop and CMD-V to paste
3. CMD-B (BOOP!) and type "upc" for UPCASE
4. Arrow key to UPCASE and hit RETURN
5. CMD-TAB to switch back to my web browser where the text area selection is still highlighted
6. CMD-V to paste the transformed item

Basically save two CMD keystrokes (CMD-A and CMD-C in Boop) after step 4 by offering this as a preference.

**Comment by IvanMathy** [comment]
Hey there!

Thanks for the suggestion; it's an interesting idea. My first thought would be to add it as a shortcut, by holding Shift when pressing enter from the picker for example. Would that be helpful?

The question is: what should happen when you only have part of the text selected? Or when you have multiple shorter selections?

I'd love to hear your thoughts on this!

**Comment by jpollone** [comment]
Oh, I think that's brilliant–add the modifier key when selecting from the picker to invoke a copy. 

The thought is that I pasted the an entire line or snippet for a reason, so after a partial or full transformation of that buffer, the entire buffer should be copied to the clipboard. After a transformation (full or partial), I'll switch to my other app (CMD-tab) and paste directly over the text that is still selected.
